### ISSUE TITLE
Switch and Buzzer Entity disabled DishWaser

### ISSUE THREAD

**Issue by Benriverato** [Andre0512/hon#83]
**Describe the bug**
The switch On/Off Entity and Buzzer it's disabled always. 

**To Reproduce**
Go to Settings -> Devices -> Haier Hon -> 1 Device and control sections

**Expected behavior**
Whether the device is online or not, these entities always come Unavailable

**Screenshots**
![imagen](https://github.com/Andre0512/hon/assets/102724368/b04bf884-e2fe-4528-967d-30839d70daa3)

**Environment (please complete the following information):**
 - Home Assistant Version: 2023.6.3
 - hOn Integration Version v0.9.0-beta.5
 - pyhOn Version 

**Additional context**
**Device Log**
[DeviceLog.txt](https://github.com/Andre0512/hon/files/11865177/DeviceLog.txt)

**Data Archive**
[dw_509.zip](https://github.com/Andre0512/hon/files/11865188/dw_509.zip)

**Comment by Andre0512** [comment]
Hi, I checked your data and everything looked right so far. If the dish washer turns on and gets set in the remote control mode, the buttons should become available. :thinking: 
Can you start a program with the hOn app? (while the switches are unavailable)

**Comment by Benriverato** [comment]
> Hi, I checked your data and everything looked right so far. If the dish washer turns on and gets set in the remote control mode, the buttons should become available. 🤔
> Can you start a program with the hOn app? (while the switches are unavailable)

Yes, I can manage the dishwasher from the hOn app, as well as see the statuses and the remaining washing time both in Home assistant and in the App. But those entities are always unavailable

**Comment by Andre0512** [comment]
Okay strange 🤔

![2023-06-29 10_31_28-Bedienungsanleitung Haier XIB 6B2D3FB Geschirrspüler](https://github.com/Andre0512/hon/assets/20325077/e55fc93a-ad45-40ac-bc36-cb237cff1115)
Just to be absolute clear we talk about the same thing, you set this on your machine?

**Comment by Benriverato** [comment]
> Okay strange 🤔
> 
> ![2023-06-29 10_31_28-Bedienungsanleitung Haier XIB 6B2D3FB Geschirrspüler](https://user-images.githubusercontent.com/20325077/249717905-e55fc93a-ad45-40ac-bc36-cb237cff1115.png) Just to be absolute clear we talk about the same thing, you set this on your machine?

Yes, I do those steps until I see Connected on the display. I can manage it from the App. When it is connected in Home Assistant, the entity of the dishwasher appears as Connected, but the 2 aforementioned entities appear as not available.

**Comment by anthfarr** [comment]
@Benriverato do your sensors also become unavailable?
I have a similar problem with my dishwasher (different model) but I get everything as unavailable.

It's probably a different issue, just wanted to check first
